.github/workflows/test.yml

```diff
@@ -180,12 +190,13 @@ jobs:
           }
           kubectl get pods --all-namespaces
 
-      - name: Run example script
+      - name: Maven Build
+        run: mvn -Dmaven.test.skip=true clean install
```

Comment:
If we use clean here you're not using the cache at all, so or delete the cache or remove the clean.

Reply:
Thanks! `clean` removed